CHANGES.rst

```diff
@@ -16,6 +16,7 @@ Unreleased
     :issue:`2188`
 -   Redirects with the test client set ``RAW_URI`` and ``REQUEST_URI``
     correctly. :issue:`2151`
+-   ``TestResponse`is no longer collected by ``pytest``. :issue:`2317`
```

Comment:
I don't think this needs a changelog entry; it's not relevant for people using werkzeug

Reply:
Understood

Reply:
Done!